packages/admin/src/pages/super-admin-dashboard/user/[id]/delete-user.page.tsx

```diff
@@ -0,0 +1,82 @@
+import {
+  FlexibleQuestionPageLayout,
+  QuestionPageGetServerSideProps,
+} from 'gap-web-ui';
+import { GetServerSidePropsContext } from 'next';
+import CustomLink from '../../../../components/custom-link/CustomLink';
+import Meta from '../../../../components/layout/Meta';
+import InferProps from '../../../../types/InferProps';
+import { getUserTokenFromCookies } from '../../../../utils/session';
+import {
+  deleteUserInformation,
+  getUserById,
+} from '../../../../services/SuperAdminService';
+
+export async function getServerSideProps(context: GetServerSidePropsContext) {
+  const userId = context.params?.id as string;
+
+  async function handleRequest(_body: { _csrf: string }, jwt: string) {
+    deleteUserInformation(userId, jwt);
+  }
+
+  async function fetchPageData(jwt: string) {
+    const user = await getUserById(userId, jwt);
+    return {
+      user,
+      userId,
+    };
+  }
+
+  return QuestionPageGetServerSideProps({
+    context,
+    fetchPageData,
+    handleRequest,
+    jwt: getUserTokenFromCookies(context.req),
+    onErrorMessage: 'Failed to delete user, please try again later.',
+    onSuccessRedirectHref: `/super-admin-dashboard/`,
+  });
+}
+
+const DeleteUserPage = ({
+  formAction,
+  pageData,
+  csrfToken,
+  fieldErrors,
+}: InferProps<typeof getServerSideProps>) => {
+  return (
+    <>
+      <Meta title="Delete Department" />
```

Comment:
should this say user instead of department?

Reply:
Fixed in latest commit